plugins/sync-font/client.js

```diff
@@ -1,5 +1,5 @@
 export function clientSyncFont(client) {
 	return client.request('font').then(font => {
-		document.documentElement.style.fontSize = font.size;
+		document.documentElement.style.fontSize = font.sizeRoot ?? font.size;
```

Comment:
Kinda thinking this might want to be
```js
if (document.body.classList.contains('d2l-typography')) {
  document.documentElement.style.fontSize = font.sizeRoot ?? font.size;
} else {
  document.documentElement.style.fontSize = font.size;
}
```

Reply:
> @grant-cleary I'm more concerned that people are relying on the font size being smaller then it should be in current FRA's (root html font size being `17.1px` vs `20px`). This current change will mean that font sizes will "get bigger" if `d2l-typography` isn't applied to the FRA's body class. Could cause stylistic changes that aren't desired. Although that would only happen if they update the `ifrau` client in their FRA.

Ah, fair. Yeah, I'm always a bit less concerned when teams need to make a conscious choice to upgrade, as (hopefully) they would look at the changes to figure out whether this would impact them. But you're right that in that case this would be an impactful change, so I'm also not at all opposed to your proposed solution of checking specifically for d2l-typography as a safeguard.